src/Utils/Helpers.ts

```diff
@@ -233,6 +233,15 @@ export class Helpers {
     }, {});
   }
 
+  // Error type checkers
+  public static getErrorMessage(e: unknown): string {
```

Comment:
I do not see the reason to use this helper

Reply:
Ok, I can move types checking to catch blocks.
But, there are a lot of places where we should use that check.